src/ray/raylet/node_manager.cc

```diff
@@ -1319,7 +1335,7 @@ void NodeManager::AssignTask(Task &task) {
           // worker once one becomes available.
           // (See design_docs/task_states.rst for the state transition diagram.)
           local_queues_.QueueReadyTasks(std::vector<Task>({task}));
-          DispatchTasks();
+          DispatchTasks(nullptr);
```

Comment:
Like we were saying earlier, I think this and the `QueueReadyTasks` line can be replaced with `AssignTask(task)`.

Reply:
Yes, just did it and testing.